ethcore/sync/src/chain/mod.rs

```diff
@@ -1083,6 +1096,11 @@ impl ChainSync {
 			self.peers.values_mut().nth(peer).map(|peer_info|
 				peer_info.last_sent_transactions.clear()
 			);
+
+			//re-broadcast private packets as well
+			self.peers.values_mut().nth(peer).map(|peer_info|
+				peer_info.reset_private_stats()
```

Comment:
Why do we take `peers.values_mut().nth()` again, we have it just above. Please add a second line to `map`

Reply:
Moved to the proper place